routes/index.js

```diff
@@ -179,14 +179,7 @@ module.exports = () => {
 						_profile.bio = account.bio;
 
 						Account.findOne({ username: requestingUser }).then(acc => {
-							if (TRIALMODS.includes(requestingUser)) {
-								try {
-									_profile.lastConnectedIP = '-' + obfIP(_profile.lastConnectedIP);
-								} catch (e) {
-									_profile.lastConnectedIP = 'something went wrong';
-									console.log(e);
-								}
-							} else if (!acc || !acc.staffRole || acc.staffRole.length === 0 || acc.staffRole === 'contributor') {
+							if (!acc || !acc.staffRole || acc.staffRole.length === 0) {

```

Comment:
`if (!acc || !acc.staffRole || acc.staffRole.length === 0) {`

3rd one is the same as 2nd one in javascript and can be removed.

Reply:
roger that

Reply:
got this i think unless my count is off.. got rid of 
` || acc.staffRole.length ===0 ` right ?